pilot/pkg/bootstrap/server.go

```diff
@@ -565,14 +565,17 @@ func (s *Server) initIstiodAdminServer(args *PilotArgs, wh *inject.Webhook) erro
 		Addr:    args.ServerOptions.HTTPAddr,
 		Handler: s.httpMux,
 	}
+	if args.SecuredDebug {
+		s.httpServer.Addr = "127.0.0.1:8080"
```

Comment:
I would have expected the HTTPAddr default to just be changed to `localhost:8080`. Is there a reason to prefer this?

Reply:
fixed!